.travis.yml

```diff
@@ -4,7 +4,7 @@ services:
 - docker
 language: go
 go:
-- 1.9
+- 1.10.4
```

Comment:
I'd suggest you may want this as `"1.10.x"` which would allow this to be the latest 1.10.x package

Reply:
Oh, wasn’t aware of the quotes trick.
That sounds like exactly what I need. Thanks!